index.md

```diff
@@ -29,71 +29,79 @@ $$
 
 ```python
-# Define domain
+model = gmsh.model()
+model.add("Channel")
+model.setCurrent("Channel")
+
 h = 0.25
 r = 0.3*h
-box = Box(Point(0, 0, 0), Point(1, h, h))
-s0 = Sphere(Point(0.3, 0.50*h, 0.50*h), r)
-s1 = Sphere(Point(0.5, 0.65*h, 0.65*h), r)
-s2 = Sphere(Point(0.7, 0.35*h, 0.35*h), r)
-domain = box - s0 - s1 - s2
-
-# Generate mesh
-mesh = generate_mesh(domain, 32)
+box = model.occ.addBox(0, 0, 0, 1, h, h)
+s0 = model.occ.addSphere(0.3, 0.50*h, 0.50*h, r)
+s1 = model.occ.addSphere(0.5, 0.65*h, 0.65*h, r)
+s2 = model.occ.addSphere(0.7, 0.35*h, 0.35*h, r)
+
+domain = model.occ.cut([(3, box)], [(3, s0), (3, s1), (3, s2)])
+
+model.occ.synchronize()
+model.mesh.generate(3)
 ```
 
 ## High-performance computing
 
 ![An example image](/assets/img/tc_vm.png){: .image-left }
-Each component of the FEniCS platform has been fundamentally designed for parallel processing. 
-Executing a FEniCS script in parallel is as simple as calling `mpirun -np 64 python script.py`. 
+Each component of the FEniCSx platform has been fundamentally designed for parallel processing. 
+Executing a FEniCSx script in parallel is as simple as calling `mpirun -np 64 python script.py`. 
 This framework allows for rapid prototyping of finite element formulations and solvers on 
 laptops and workstations, and the same code may then be deployed on large high-performance 
 computers.
 
-The figure shows the von Mises stresses computed from a nonlinear thermomechanical FEniCS 
+The figure shows the von Mises stresses computed from a nonlinear thermomechanical FEniCSx 
 simulation of a turbocharger. The finite element system of linear equations comprises more 
 than \\(3.3 \times 10^9\\) degrees of freedom. The solver was initially developed on a desktop computer 
 for a small scale problem, and the same code was then deployed on a supercomputer using over 
-24000 parallel processes.
+24,000 parallel processes.
 
 ## Installation and documentation
 
-FEniCS is available for a range of platforms (Linux, Mac, Windows). Choose between Docker 
-containers, binary packages and source code. Visit our [installation page](download/index.md) to get the latest 
-version of FEniCS. FEniCS comes with [extensive documentation](documentation/index.md) and numerous examples. A good 
-starting point is the [FEniCS Tutorial](documentation/tutorial.md).
+FEniCSx is available for a range of platforms (Linux, Mac, Windows). Choose between Docker 
+containers, binary packages, Spack packages and source code. Visit our [installation page](download/index.md) to get the latest 
+version of FEniCSx. FEniCSx comes with [extensive documentation](documentation/index.md) and numerous examples. A good 
+starting point is the [FEniCSx Tutorial](https://jorgensd.github.io/dolfinx-tutorial/).
 
 ## About
-The FEniCS Project is developed and maintained as a freely available, open-source project by a 
+The FEniCSx Project is developed and maintained as a freely available, open-source project by a 
```

Comment:
I think this one might not need the x, as it's still the FEniCS Project?

Reply:
This was my thought too.